docs/JSON-RPC.md

```diff
@@ -129,6 +129,23 @@ Results:
 | result.version | string | The Jamulus version. |
```

Comment:
Consistency on a page is a mercy for readers, and this page consistently puts a period behind non-sentences. Ideally, none would include a period.